docs/reference/settings/ssl-settings.asciidoc

```diff
@@ -26,12 +26,12 @@ Defaults to +{client-auth-default}+.
 endif::client-auth-default[]
 endif::server[]
 
-ifdef::verifies[]
 +{ssl-prefix}.ssl.verification_mode+::
 (<<static-cluster-setting,Static>>)
-Controls the verification of certificates.
+Defines how to verify the certificates presented by another party in the TLS 
+connection. This setting accepts the following values for TLS _clients_, but 
+should not be set for TLS _servers_:
```

Comment:
This assumes that the reader understands what _clients_ and _servers_ means with enough depth to know when they should/shouldn't make use of it. I don't think experience supports that assumption.

Something more along these lines (I didn't do any word-smithing here, it's totally a first draft)

```suggestion
ifdef::verifies[]
The SSL settings in `{ssl-prefix}.ssl` control a _server context_ for TLS. The use of `verification_mode` in a TLS _server_ is discouraged. 
endif::verifies[]
Defines how to verify the certificates presented by another party in the TLS 
connection:
```

Reply:
>The SSL settings in `{ssl-prefix}.ssl` control a _server context_ for TLS. The use of `verification_mode` in a TLS _server_ is discouraged. 

If we're fairly certain that users don't know what _servers_ refers to, do we anticipate that our users know what a _server context_ is? Perhaps we need to provide some additional context 😆 Sorry, bad joke. Perhaps something like:

>The SSL settings in `{ssl-prefix}.ssl` control a _server context_ for TLS, which defines the settings for the TLS connection. The use of `verification_mode` in a TLS _server_ is discouraged.

Reply:
@tvernum What do you think about Adam's suggestion? Let's get this PR merged... :)

Reply:
I'm happy with Adam's suggestion. My main intention is that `ifndef::verifies[]`, we include some sort of warning that says "you don't really want to do this". That is, we don't assume that the reader knows whether an SSL context is "client" or "server" - we explicitly tell them whenever it's a server context and steer them away from this setting.